next.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  extends: [
+    "@open-turo/eslint-config-typescript",
+    "plugin:@next/next/recommended",
+  ],
+  rules: {
+    "import/no-default-export": "off",
+    "import/no-extraneous-dependencies": [
+      "error",
+      { devDependencies: ["test/**/", "**/*.test.*"] },
+    ],
+    "node/no-unpublished-import": [
+      "error",
+      {
+        allowModules: [
+          "@graphql-typed-document-node/core",
```

Comment:
I think I would delete this one, as graphql is not tied directly to react / next but is something more particular to a project, but ok to keep it too